bio-formats/downloads/index.html

```diff
@@ -22,7 +22,7 @@ <h3 class="text-center">Tools</h3>
                         <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_bio-formats.svg">
                         <h5>Bio-Formats Package</h5>
                         <h6>bioformats_package.jar</h6>
-                        <p class="card-caption">The complete bundle for end users containing everything needed to read images into ImageJ. Instructions for installing Bio-Formats in ImageJ are available in the <a href="https://docs.openmicroscopy.org/bio-formats/{{ site.bf.version }}/users/imagej/installing.html" taget="_blank">user guide</a>.</p>
+                        <p class="card-caption">The complete bundle for end users containing everything needed to read images into ImageJ. Instructions for installing Bio-Formats in ImageJ are available in the <a href="https://bio-formats.readthedocs.io/en/v{{ site.bf.version }}/users/imagej/installing.html" taget="_blank">user guide</a>.</p>
```

Comment:
Not introduced by this PR, but should `taget="_blank"` be switched to `target="_blank"`?